packages/hydroxide/build.sh

```diff
@@ -11,6 +12,7 @@ termux_step_make() {
 	termux_setup_golang
 	export GOPATH=$TERMUX_PKG_BUILDDIR
 
+	patch -u $TERMUX_PKG_SRCDIR/imap/mailbox.go -i mailbox.go.patch
```

Comment:
No. You don't need to modify `build.sh` other than revision.

I mean `mailbox.go.patch` lacks the following header:

```
--- a/imap/mailbox.go
+++ b/imap/mailbox.go
```